parm/config/config.aero

```diff
@@ -13,6 +13,9 @@ case $machine in
   "ORION")
     AERO_INPUTS_DIR="/work2/noaa/global/wkolczyn/noscrub/global-workflow/gocart_emissions"
     ;;
+  "S4")
+    AERO_INPUTS_DIR="/data/prod/glopara/gocart_emissions"
```

Comment:
Are you keeping this sync'd with the Hera copy?

Reply:
I sync'd it two weeks ago, but haven't been keeping up since then.  I will add it to the nightly rsync, though.  Thanks.

Reply:
Cool...this set doesn't update too much (same with fix) so maybe a weekly check is sufficient (or after an announcement) but I leave that up to you. Recommend using the `--checksum` flag with rsync from time to time to catch any diffs that aren't caught by the usual checks rsync does on metadata and filesize. Using that flag significantly increases rsync time though so not needed all the time.